src/index.html

```diff
@@ -13,7 +14,7 @@
         <div class="p-2">
             <div class="listContent">
                 <div class="row">
-                    <div class="col-md-12"">
+                    <div class="col-md-12">
                         <div class="col-md-8">
                             <h3>Today's To DO</h3>
```

Comment:
- [Optional] Consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)